lib/puppet/transaction/report.rb

```diff
@@ -183,7 +194,7 @@ def initialize(kind, configuration_version=nil, environment=nil, transaction_uui
     @host = Puppet[:node_name_value]
     @time = Time.now
     @kind = kind
-    @report_format = 5
+    @report_format = 6
```

Comment:
@kbarber and yeah, we've broken it a lot in the past. It resulted in chaos, so I made a small tantrum about how let's not do that anymore. :)